spec/features/04_movies_create_spec.rb

```diff
@@ -0,0 +1,205 @@
+require "rails_helper"
+
+feature "Create Movies" do
+  context "As an authenticated user" do
+    before do
+      user1 = User.create(
+        first_name: "John",
+        last_name: "Doe",
+        username: "johndoe",
+        email: "john@doe.com",
+        password: "abcdef1",
+        newsletter: true
+      )
+      visit "/"
```

Comment:
you can move this into your spec_helper and make it a method. Something like `user_sign_in`